src/js/page/ui/bg-fill-button.js

```diff
@@ -16,17 +16,11 @@ export default class BgFillButton extends FloatingActionButton {
 
   onClick(event) {
     super.onClick(event);
-    // event adds a background color
-    if (this.container.classList.contains('active')) {
-      this.container.classList.remove('active');
-      this.setColor('transparent');
-    } else {
-      this.container.classList.add('active');
-      this.setColor('rgba(0, 0, 0, 0.7)');
-    }
-  }
 
-  setColor(color) {
-    document.documentElement.style.backgroundColor = color;
+    const hasActiveClass = this.container.classList.contains('active');
+    const hasDarkClass = document.documentElement.classList.contains('bg-dark');
+
+    this.container.classList.toggle('active', !hasActiveClass);
+    document.documentElement.classList.toggle('bg-dark', !hasDarkClass);
```

Comment:
```suggestion
  if (this.container.classList.contains('active')) {
    this.container.classList.remove('active');
    document.documentElement.classList.remove('bg-dark');
  } else {
    this.container.classList.add('active');
    document.documentElement.classList.add('bg-dark');
  }
```

The reason I prefer this style is it's clear what button state relates to which `<html>` background state. Unless i'm missing something in the other options, it isn't clear without knowing the _current_ state of `document.documentElement.classList`.

Reply:
Your suggested snippet relies on `active` for `bg-dark` too?

Isn't this what you prefer? https://github.com/jakearchibald/svgomg/pull/324/commits/5ed2ab80f6d9db1bad9b0fe2d2f5d8c3ef7b8c66

I kept the patches separately for this reason, so I can go back to whatever solution we want. :)